charts/incubator/tt-rss/values.yaml

```diff
@@ -0,0 +1,179 @@
+image:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+nginxImage:
+  repository: cthulhoo/ttrss-web-nginx
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f963b38ef8695d05c7f219e9405648dcb2f1f3c0b81609353bb1ba078bc8e318
+updaterImage:
+  repository: cthulhoo/ttrss-fpm-pgsql-static
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+persistence:
+  html:
+      enabled: true
+      targetSelector:
+        main:
+          main:
+            mountPath: /var/www/html
+        nginx:
+          nginx:
+            mountPath: /var/www/html
+            readOnly: true
+        updater:
+          updater:
+            mountPath: /var/www/html
+  config:
+    enabled: true
+    targetSelector:
+        main:
+          main:
+            mountPath: /opt/tt-rss/config.d
+        updater:
+          updater:
+            mountPath: /opt/tt-rss/config.d
+  # plugins:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/plugins.local
+  # themes:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/themes.local
+cnpg:
+  main:
+    enabled: true
+    user: tt-rss
+    database: tt-rss
+portal:
+  open:
+    enabled: true
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+            readiness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
```

Comment:
https://gitlab.tt-rss.org/tt-rss/helm-charts/tt-rss/-/blob/master/templates/app-deployment.yaml#L77-83
You can use this for readiness and liveness (assuming is the same image)

Reply:
Thank you! Updated in https://github.com/truecharts/charts/pull/17293/commits/a1eda14b489d9045f8136b29ee376398ae039728.